pkg/lib/operatorclient/service_test.go

```diff
@@ -0,0 +1,160 @@
+package operatorclient
+
+import (
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes/fake"
+	clienttesting "k8s.io/client-go/testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestUpdateService(t *testing.T) {
+	gvr := schema.GroupVersionResource{Group: "", Version: "v1", Resource: "services"}
+	var wildcard clienttesting.Action = clienttesting.ActionImpl{Verb: "wildcard!"}
```

Comment:
what's the reasoning behind this var? looks interesting.

Reply:
If this is referenced in one of the test cases, it matches any single call to the fake client. I wanted to avoid maintaining test expectations for the uninteresting Get calls.

Reply:
Interesting, will have to check that out. Haven't seen it before